LoyaltyProgram_Intern/src/main/java/com/pgs/soft/dto/PasswordDTO.java

```diff
@@ -0,0 +1,43 @@
+package com.pgs.soft.dto;
+
+public class PasswordDTO {
```

Comment:
Please change name to ChangePasswordRequestDTO - name should said exactly what class is responsible for